config/test.json

```diff
@@ -1,7 +1,10 @@
 {
   "storeTrigger": {
     "url": "http://127.0.0.1:8081",
-    "subUrl": "ingestion"
+    "subUrl": {
```

Comment:
Remove the suburls. We will use it hard-coded since we have more than one suburl. There is no added value to do this configurabilly

Reply:
I think it should be in the configuration and not hard codded because it is the best practice.